scaler/scaler-core/scaling-methods/linear.js

```diff
@@ -27,7 +27,15 @@ const {maybeRound} = require('../utils.js');
 function calculateSize(spanner) {
   return baseModule.loopThroughSpannerMetrics(spanner, (spanner, metric) => {
     if (baseModule.metricValueWithinRange(metric)) return spanner.currentSize;
-    else return maybeRound(Math.ceil(spanner.currentSize * metric.value / metric.threshold), spanner.units, metric.name);
+    else {
+      var suggestedSize = Math.ceil(spanner.currentSize * metric.value / metric.threshold)
+      
+      if (suggestedSize < spanner.currentSize && spanner.scaleInLimit) {
+        const limit = spanner.currentSize * (spanner.scaleInLimit/100)
```

Comment:
It would help the operator if you added a log line here that shows the original suggestion and the clamped value.

Since now the original suggestion is no longer logged